### DISCUSSION TITLE
generate-traffic.py error

### DISCUSSION THREAD

**Discussion by heria711** [carla-simulator/carla#6168] [Q&A]
my carla is built from source on windows (hardware & software satisfying requirements). After launching (default settings) , i tried to operate some official examples provided in file 'pythonapi\examples'. According to official instructions, 'dynamic-weather.py' runs successfully, while running 'generate-traffic.py', it quits immediately and shows no result, coming with (the carlaue4.exe is stuck and unresponsive). what's wrong with 'generate-traffic.py', or my opration error? Thanks a lot

**Comment by majda-22** [comment]
I have the same problem and I did not find solution  please if you have found it could you help me please

**Comment by Madecu** [comment]
Thank you for posting this discusion, we have created an issue to llok into it. We will post here when we have some uptade.
We are sorry for the delay in response, we will do our best to reduce respone times.
